src/main/scala/com/amazon/deequ/profiles/ColumnProfilerRunBuilder.scala

```diff
@@ -106,11 +108,27 @@ class ColumnProfilerRunBuilder(val data: DataFrame) {
     this
   }
 
+  /**
+   * Set KLL parameters.
+   *
+   * @param kllParameters kllParameters(sketchSize, shrinkingFactor, numberOfBuckets)
+   */
   def setKLLParameters(kllParameters: Option[KLLParameters]): this.type = {
     this.kllParameters = kllParameters
     this
   }
 
+  /**
```

Comment:
Let us use enums instead of strings for the types

Reply:
How about using the ConstrainableDataTypes which is already defined in Deequ? It is public function so that spark app could use it too.